core/meta/src/TClass.cxx

```diff
@@ -4448,46 +4448,49 @@ TVirtualStreamerInfo* TClass::GetStreamerInfo(Int_t version /* = 0 */) const
 
 TVirtualStreamerInfo* TClass::GetStreamerInfoAbstractEmulated(Int_t version /* = 0 */) const
 {
-   R__LOCKGUARD(gInterpreterMutex);
+   TVirtualStreamerInfo* sinfo = nullptr;
 
-   TString newname( GetName() );
+   TString newname(GetName());
    newname += "@@emulated";
 
-   TClass *emulated = TClass::GetClass(newname);
+   R__LOCKGUARD(gInterpreterMutex);
 
-   TVirtualStreamerInfo* sinfo = 0;
+   TClass *emulated = TClass::GetClass(newname);
 
-   if (emulated) {
+   if (emulated)
```

Comment:
Does clang-format really 'force' one line if statement to be without curly brackets?